assets/dev/scss/frontend/widgets/alert.scss

```diff
@@ -18,15 +18,31 @@
 
 	button.elementor-alert-dismiss {
 		position: absolute;
-		@include end(10px);
-		top: 10px;
+		@include end(var(--dismiss-icon-horizontal-position, 10px));
```

Comment:
Just an observation from my side. 
I had expected a space in between 'end(' and 'var'. 
I checked for similar cases in the repository and I see that other developers in 99% follow your way. 

Question: do you know why people don't punt a space? 
I would be tempted to add a space here.

Reply:
@hein-obox @louiswol94 
Good question.
There are some differences between our style rules in PHP, JS, and CSS.
Not sure what the origin of this distinction is.
In SCSS, we don't have spaces between parentheses and their contents.